src/dyncfg/src/lib.rs

```diff
@@ -299,6 +299,19 @@ impl<T: ConfigType> ConfigValHandle<T> {
     pub fn get(&self) -> T {
         T::from_val(self.val.load())
     }
+
+    /// Return a new handle that returns the constant value provided,
+    /// generally for testing.
+    pub fn disconnected<X>(value: X) -> Self
```

Comment:
We could additionally mark this with `#[cfg(test)]` if it should never be used in normal code

Reply:
Tried this, but no luck... since it ends up being used in _another_ crate's test.

To bad! Thankfully shouldn't be too much of a footgun, given how this stuff is used in practice.